src/gui.rs

```diff
@@ -193,7 +206,9 @@ pub enum Message {
 impl Default for Controls {
     fn default() -> Self {
         Self {
+            showing: Showing::Nothing,
             measure_nuance: 0.0,
+            lower_limit: f32::MIN,
```

Comment:
I prefer more explicit: `strain_lower_limit`

Reply:
proly right. on it.